data/json/Collisions18/Cert_ABCD_13TeV_Legacy2018_Collisions18_JSON.txt

```diff
@@ -0,0 +1 @@
+Cert_314472-325175_13TeV_Legacy2018_Collisions18_JSON.txt
```

Comment:
Is this attended?

Reply:
This might lead to broken PUWeightFiles

Reply:
Yest, it's just a symbolic link from `ABCD` to the whole-year JSON. It's intended to simplify things in the `do_pileup.sh` script.

Reply:
Why would it break the PUWeightFiles?

Reply:
I just thought, this might be a copy paste mistake and you tried to generate the PUWeightFile from a faulty input.